newrelic-agent/src/main/java/com/newrelic/agent/instrumentation/weaver/ClassLoaderClassTransformer.java

```diff
@@ -128,7 +128,7 @@ public ClassLoaderClassTransformer(InstrumentationProxy instrumentation, Set<Str
         // Try to use a custom Guava-based extension template to make NewField access better
         try {
             extensionTemplate = WeaveUtils.convertToClassNode(WeaveUtils.getClassBytesFromClassLoaderResource(
-                    GuavaBackedExtensionClass.class.getName(), GuavaBackedExtensionClass.class.getClassLoader()));
+                    CaffeineBackedExtensionClass.class.getName(), CaffeineBackedExtensionClass.class.getClassLoader()));
```

Comment:
`GuavaBackedExtensionClass` is now Caffeine-backend, so a rename seemed in order.